.travis.yml

```diff
@@ -97,6 +97,7 @@ script:
    [ -n "$TRAVIS_CVC4" ] && run makeCheck
    [ -z "$CVC4_SYMFPU_BUILD" ] && run makeInstallCheck && run makeExamples
    [ -z "$TRAVIS_CVC4" ] && error "Unknown Travis-CI configuration"
+   [[ "$TRAVIS_CVC4_PYTHON_BINDINGS" == "yes" ]] && $TRAVIS_PYTHON -c "import pycvc4"
```

Comment:
Can you move this line to `makeInstallCheck` and add

```
if [[ "$TRAVIS_CVC4_PYTHON_BINDINGS" == "yes" ]]; then
  $TRAVIS_PYTHON -c "import pycvc4" || exit 1
fi
```

Reply:
Thanks, good catch. Sorry, I should have read the logs more carefully.

Reply:
No worries :-)